### PR TITLE
RHDEVDOCS-3184 - How to create a secret

### PR DESCRIPTION
**Pull request by libander**
Aligned team: Dev Tools
For branches: main, 4.8+
Jira:https://issues.redhat.com/browse/RHDEVDOCS-3184
Direct link to doc preview: https://deploy-preview-42333--osdocs.netlify.app/openshift-enterprise/latest/logging/cluster-logging-external.html#creating-a-secret
SME review: @alanconway
QE review: @anpingli
Peer review: @rolfedh

### CODE REVIEW

**Comment by anpingli** [comment]
@libander 
I think it is better to include the username and password in this example:
oc create secret generic -n openshift-logging mya-secret   --from-file=tls.key=your_key_file   --from-file=tls.crt=your_crd_file   --from-file=ca-bundle.crt=your_bundle_file --from-literal=username=your_username --from-literal=password=your_password

**Comment by libander** [comment]
> @libander I think it is better to include the username and password in this example: oc create secret generic -n openshift-logging mya-secret --from-file=tls.key=your_key_file --from-file=tls.crt=your_crd_file --from-file=ca-bundle.crt=your_bundle_file --from-literal=username=your_username --from-literal=password=your_password

Ok thank you!

File: modules/cluster-logging-collector-log-forwarding-about.adoc [hunk 1]
```diff
@@ -177,3 +177,21 @@ If missing, SASL is automatically enabled when any of the other `sasl.` keys are
 * `sasl.mechanisms`: (array) List of allowed SASL mechanism names.
 If missing or empty, the system defaults are used.
 * `sasl.allow-insecure`: (boolean) Allow mechanisms that send clear-text passwords. Defaults to false.
+
+== Creating a Secret
+
+You can create a secret from the directory containing your certificate and key files using the following command:
```
Comment (rolfedh) [inline modules/cluster-logging-collector-log-forwarding-about.adoc:183]:
```suggestion
You can create a secret from the directory containing your certificate and key files by using the following command:
```

File: modules/cluster-logging-collector-log-forwarding-about.adoc [hunk 2]
```diff
@@ -177,3 +177,21 @@ If missing, SASL is automatically enabled when any of the other `sasl.` keys are
...
 If missing or empty, the system defaults are used.
 * `sasl.allow-insecure`: (boolean) Allow mechanisms that send clear-text passwords. Defaults to false.
+
+== Creating a Secret
+
+You can create a secret in the directory that contains your certificate and key files by using the following command:
+
+----
+$ oc create secret generic -n openshift-logging my-secret \
+ --from-file=tls.key=your_key_file
```
Comment (JStickler) [inline modules/cluster-logging-collector-log-forwarding-about.adoc:187]: Per the style guide, you should wrap any replaceable values in angle brackets (<>) .  So this should be:

--from-file=tls.key=<your_key_file>

(And the following lines as well)

https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output

**Review by JStickler — CHANGES_REQUESTED** [review]
Needs a small formatting change to conform to the style guide.

File: modules/cluster-logging-collector-log-forwarding-about.adoc [hunk 3]
```diff
@@ -177,3 +177,21 @@ If missing, SASL is automatically enabled when any of the other `sasl.` keys are
 * `sasl.mechanisms`: (array) List of allowed SASL mechanism names.
 If missing or empty, the system defaults are used.
 * `sasl.allow-insecure`: (boolean) Allow mechanisms that send clear-text passwords. Defaults to false.
+
+== Creating a Secret
+
+You can create a secret in the directory that contains your certificate and key files by using the following command:
+
```
Comment (rolfedh) [inline modules/cluster-logging-collector-log-forwarding-about.adoc:184]:
```suggestion
[subs="+quotes"]
```

File: modules/cluster-logging-collector-log-forwarding-about.adoc (hunk 2 above)
Comment (rolfedh) [reply modules/cluster-logging-collector-log-forwarding-about.adoc:187]: Thanks, @JStickler. Sorry I missed that, @libander. The guidelines for doing this are here:  https://redhat-documentation.github.io/supplementary-style-guide/#user-replaced-values

File: modules/cluster-logging-collector-log-forwarding-about.adoc [hunk 4]
```diff
@@ -177,3 +177,21 @@ If missing, SASL is automatically enabled when any of the other `sasl.` keys are
...
+----
+$ oc create secret generic -n openshift-logging my-secret \
+ --from-file=tls.key=your_key_file
+ --from-file=tls.crt=your_crd_file
+ --from-file=ca-bundle.crt=your_bundle_file
+ --from-literal=username=your_username
+ --from-literal=password=your_password
```
Comment (rolfedh) [inline modules/cluster-logging-collector-log-forwarding-about.adoc:191]:
```suggestion
$ oc create secret generic -n openshift-logging _<my-secret>_ \
 --from-file=tls.key=_<your_key_file>_
 --from-file=tls.crt=_<your_crd_file>_
 --from-file=ca-bundle.crt=_<your_bundle_file>_
 --from-literal=username=_<your_username>_
 --from-literal=password=_<your_password>_
```

**Review by JStickler — APPROVED** [review]
LGTM

**Review by anpingli — COMMENTED** [review]
/lgtm

**Review by rolfedh — APPROVED** [review]

**Review by alanconway — APPROVED** [review]
/lgtm

**Pull request merged**